sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1595,8 +1595,8 @@ def callback_mock(renewable, error):
         auto_lock_renew = AutoLockRenewer()
         auto_lock_renew._renew_period = 1
         with auto_lock_renew: # Check that in normal operation it does not get called
-            auto_lock_renew.register(receiver, renewable=MockReceivedMessage(), on_lock_renew_failure=callback_mock)
-            time.sleep(3)
+            auto_lock_renew.register(receiver, renewable=MockReceivedMessage(lock_duration=5), on_lock_renew_failure=callback_mock)
```

Comment:
2 seconds would fail randomly on windows, increase the time a bit to let AutoLockRenewer to have enough time for renewing work.

also, the minimum allowed lock duration by the service is 5 seconds